src/pages/Application/Application.page.tsx

```diff
@@ -166,8 +166,8 @@ export const ApplicationPage = () => {
                 application.participatingAs === "Hacker"
                     ? hackerSpecificValidation
                     : application.participatingAs === "Mentor"
-                    ? mentorSpecificValidation
-                    : volunteerSpecificValidation;
+                      ? mentorSpecificValidation
+                      : volunteerSpecificValidation;
```

Comment:
Note: someone should run prettier on the entire project... avoids these formatting changes.

Reply:
We plan to move to biome.js so haven't done the formatting. Should write workflows to check proper code format and lint before allowing merges. We'll work on it.